Formula/openfast.rb

```diff
@@ -0,0 +1,27 @@
+class Openfast < Formula
+  desc "NREL-supported OpenFAST whole-turbine simulation code"
+  homepage "https://openfast.readthedocs.io"
+  url "https://github.com/rafmudaf/openfast/archive/v2.3.0.tar.gz"
```

Comment:
This shouldn't reference a fork.

Reply:
The main repo doesn't have a v2.3.0 yet but this latest version has some bug fixes that enable it to be compiled and tested with homebrew. I've made this release on my fork and switched the repository in the formula just for testing purposes. We are working on getting the release ready in parallel to my building out the formula, so it will be switched to the main repository soon.